### PR TITLE
Count Inversion algorithm is added

### LINKED ISSUE #921: count inversion algorithm in javascript
i want to add a count inversion in an array using merge sort in javascript. please assign me this task.

### PR DESCRIPTION
**Pull request by harshitagupta11**
Fixes #921  

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
Count inversion algorithm using merge sorts added.

#### Languages Used:
- Javascipt

#### Files Added:
- Count_Inversion.js

### CODE REVIEW

**Comment by marwahmanbir** [comment]
Mention the issue number that this PR resolves, in the template above.

**Comment by harshitagupta11** [comment]
Done.

**Comment by marwahmanbir** [comment]
You have mentioned the PR number instead of issue number.

File: Count_Inversion/Count_Inversion.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    function merge(left, right) {
+        var result = [];
+        var llen = left.length;
+        var rlen = right.length;
+        var l = 0;
+        var r = 0;
+        while (l < llen && r < rlen) {
+            if (left[l] <= right[r]) {
+                result.push(left[l++])
+
```
Comment (sridharjajoo) [inline Count_Inversion/Count_Inversion.js]: extra line

File: Count_Inversion/Count_Inversion.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    }
+    //var a = [2, 4, 1];
+    /*for (var i = 0; i < 10; i++) {
+        a[i] = Math.floor(Math.random() * 100 + 1);
+    }*/
+    //console.log(a);
+    mergesort(arr);
+    return c;
+
```
Comment (sridharjajoo) [inline Count_Inversion/Count_Inversion.js]: extra line

File: Count_Inversion/Count_Inversion.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        if (len < 2) {
+            return arr;
+        }
+        var mid = Math.floor(len / 2);
+        var left = arr.slice(0, mid);
+        var right = arr.slice(mid);
+        //console.log(left);
+        //console.log(right);
+        return merge(mergesort(left), mergesort(right));
+
```
Comment (SakshiShreya) [inline Count_Inversion/Count_Inversion.js]: Do not add extra line before closing bracket.

File: Count_Inversion/Count_Inversion.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    }
+    //var a = [2, 4, 1];
+    /*for (var i = 0; i < 10; i++) {
+        a[i] = Math.floor(Math.random() * 100 + 1);
+    }*/
+    //console.log(a);
+    mergesort(arr);
+    return c;
+
+}
```
Comment (marwahmanbir) [inline Count_Inversion/Count_Inversion.js]: Add sample input-output at the end.
Add a newline after sample input-output (at the end of file).

**Pull request merged**